redis/src/types.rs

```diff
@@ -1999,7 +2080,6 @@ macro_rules! from_vec_from_redis_value {
 
 from_vec_from_redis_value!(<T> Vec<T>);
 from_vec_from_redis_value!(<T> std::sync::Arc<[T]>);
-from_vec_from_redis_value!(<T> Box<[T]>; Vec::into_boxed_slice);
```

Comment:
why prefer your implementation?

Reply:
I wanted to remove `from_vec_from_redis_value!` macro and use just `forwarded_impl!((T), std::sync::Arc<[T]>, std::sync::Arc::new);` but I wasn't able to make it compile. If you have suggestion how to make it compile this can benefit in one macro less.

Reply:
I can try, but your macro doesn't cover `Vec`, so I don't see how we can avoid having 2 macros here.

Reply:
The `from_vec_from_redis_value!` will be used only for `Vec<T>` so no need to keep macro just direct impl for Vec.

Reply:
Was able to remove from_vec_from_redis_value! macro with [`f91f8e0` (#1088)](https://github.com/redis-rs/redis-rs/pull/1088/commits/f91f8e0e7fefbd4e070a9fccf48f16c385d9c2bd) commit